docs/source/en/optimization/fp16.mdx

```diff
@@ -251,6 +253,10 @@ image = pipe(prompt).images[0]
 This feature requires `accelerate` version 0.17.0 or larger.
 </Tip>
 
+**Note**: `enable_model_cpu_offload()` is a stateful operation that installs hooks on the models and state on the pipeline. In order to properly re-offload
```

Comment:
```suggestion
**Note**: `enable_model_cpu_offload()` is a stateful operation that installs hooks on the models and state on the pipeline. In order to properly offload
```

(I might be mistaken, didn't understand what `re-offload` was referring to)

Reply:
re-offload meaning move the weights are moved back to the cpu